scripts/cleanup_edomain_templates.py

```diff
@@ -62,7 +62,7 @@ def is_ovirt_engine_running(rhevm_ip, sshname, sshpass):
         return False
 
 
-def change_edomain_state(api, state, edomain):
+def change_edomain_state(api, state, edomain, provider):
```

Comment:
@psav that was just a string. In the new commit changed that to pass the provider_obj and fetch the api, provider_name from that.